pkg/apis/deployment/v1/server_group.go

```diff
@@ -119,7 +119,7 @@ func (g ServerGroup) DefaultTerminationGracePeriod() time.Duration {
 // IsStateless returns true when the groups runs servers without a persistent volume.
 func (g ServerGroup) IsStateless() bool {
 	switch g {
-	case ServerGroupCoordinators, ServerGroupSyncMasters, ServerGroupSyncWorkers:
+	case ServerGroupSyncMasters, ServerGroupSyncWorkers:
```

Comment:
Change related to coordinator upgrade flow - it do not keep local storage, but still run upgrade operation on data in cluster